mw-webapp/src/utils/UnicodeSymbols.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Unicode symbols
+ */
+export enum UnicodeSymbols {
+
+  /**
+   * Zero-width space by unicode
+   */
+  ZERO_WIDTH_SPACE = "\u200B",
+
+  /**
+   * Dividing point by unicode
```

Comment:
If you should repeat something everywhere - just declare it in the right place once)

Reply:
Deleted